src/gryphlifehandler.js

```diff
@@ -0,0 +1,117 @@
+import * as http from "http";
+import * as ExcelJS from "exceljs";
```

Comment:
ExcelJS is not the library used to modify spreadsheet files. We use the [xlsx](https://www.npmjs.com/package/xlsx) package to modify our files. This is the format our spreadsheets are stored in so it's more viable to use this package over ExcelJS.